README.md

```diff
@@ -40,35 +41,44 @@ make dmctl      # build dmctl
 
 When DM is built successfully, you can find binaries in the `bin` directory.
 
-## Run Test
+## Run tests
 
-Run all tests, including unit test and integration test
+Run all tests, including unit tests and integration tests:
 
 ```bash
 make test
 ```
 
 ## Installing
 
-Please reference to "Deploy" section in [our doc](https://docs.pingcap.com/tidb-data-migration/)
+See the "Deploy" section in [our doc](https://docs.pingcap.com/tidb-data-migration/)
 
 ## Config File
 
-Please reference to "Configuration" section in [our doc](https://docs.pingcap.com/tidb-data-migration/dev/config-overview)
+See the "Configuration" section in [our doc](https://docs.pingcap.com/tidb-data-migration/dev/config-overview)
 
 ## Roadmap
 
 Read the [Roadmap](roadmap.md).
 
-
 ## Contributing
+
 Contributions are welcomed and greatly appreciated. See [CONTRIBUTING.md](./CONTRIBUTING.md)
 for details on submitting patches and the contribution workflow.
 
 Any questions? Let's discuss in [#sig-migrate in Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=github_sig)
 
 ## License
-DM is under the Apache 2.0 license. See the [LICENSE](./LICENSE) file for details.
 
+DM is under the Apache 2.0 license. See the [LICENSE](./LICENSE) file for details.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fpingcap%2Fdm.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fpingcap%2Fdm?ref=badge_large)
+
+## More resources
+
+- TiDB blog
+
+    - [English](https://pingcap.com/blog/)
+    - [简体中文](https://pingcap.com/blog-cn/)
+
+- [TiDB Monthly](https://pingcap.com/weekly/
```

Comment:
```suggestion
- [TiDB Monthly](https://pingcap.com/weekly/)
```

and please check it's monthly or weekly

Reply:
It's "monthly," but the URL is the weekly link.